test/e2e/conformance/cross_logical_cluster_list_test.go

```diff
@@ -270,12 +272,13 @@ func TestBuiltInCrossLogicalClusterListPartialObjectMetadata(t *testing.T) {
 
 	org := framework.NewOrganizationFixture(t, server)
 
-	cfg := server.DefaultConfig(t)
+	cfg := server.BaseConfig(t)
+	rootShardCfg := server.RootShardConfig(t)
 
 	kcpClusterClient, err := kcpclientset.NewClusterForConfig(cfg)
 	require.NoError(t, err, "failed to construct kcp client for server")
 
-	rootCfg := framework.ShardConfig(t, kcpClusterClient, "root", cfg)
+	rootCfg := framework.ShardConfig(t, kcpClusterClient, "root", rootShardCfg)
```

Comment:
can we drop this line?

Reply:
ha, stumbled upon this as well. this is used once in a wildcard list call: https://github.com/kcp-dev/kcp/blob/8c833d4f55313383b8d8bc3d4069f06822df893d/test/e2e/conformance/cross_logical_cluster_list_test.go#L306

Reply:
rootCfg will be the same as rootShardCfg. ShardConfig on rootShardCfg is a noop.

Reply:
removed